Source/Scene/Expression.js

```diff
@@ -1198,7 +1198,7 @@ define([
         //>>includeStart('debug', pragmas.debug);
         throw new DeveloperError('Operator "-" requires a vector or number argument. Argument is ' + left + '.');
```

Comment:
Sounds good.

In general, while Cesium is aggressive with developer errors in API calls, anything that deals with externally loading data should normally use `RuntimeError` instead (we need to do better about this all around).